docs/user/external_interactions.md

```diff
@@ -0,0 +1,9 @@
+# External Interactions
+
+This document describes external dependencies and prerequisites for this App to operate, including system requirements, API endpoints, interconnection or integrations to other applications or services, and similar topics.
+
+## External System Integrations
+
```

Comment:
Fill or remove

Reply:
@DistantVoyager This is up to package maintainers pls.

Reply:
Not needed. Will update PR to remove `external_interactions.md`